components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -338,6 +338,7 @@ public enum ErrorMessages {
                 "requirements of the selected registration option."),
         ERROR_CODE_INVALID_REGISTRATION_OPTION("USR-10007", "Invalid registration option."),
         ERROR_CODE_MULTIPLE_REGISTRATION_OPTIONS("USR-10008", "Multiple registration options are not supported."),
+        ERROR_CODE_INVALID_DOMAIN("USR-10009", "The email domain is not match with organization email domain."),
```

Comment:
Shall we refine the wording bit?

Reply:
The email domain does not match the organization's email domain.

Reply:
Fixed by - https://github.com/wso2-extensions/identity-governance/pull/813/commits/da9ce97ff9cdc7b0df161dc63072353d943edfc1